.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        rust: [nightly, beta, stable]
```

Comment:
I think we just need stable since we already choose to develop on it.

Reply:
Yeah, doesn't really hurt tbh. It's cached so they are all very fast and it gives us an idea if everything is compatible with the future stable versions.